src/core/content/util.ts

```diff
@@ -747,8 +766,13 @@ export function injectScriptIntoDocument(scriptUrl: string): void {
  * @param logType - Log type
  */
 /* istanbul ignore next */
-export function debugLog(text: unknown, logType: DebugLogType = 'log'): void {
+export async function debugLog(text: unknown, logType: DebugLogType = 'log'): Promise<void> {
 	const logFunc = console[logType];
+	const loggingEnabled = await getGlobalOption('debugLoggingEnabled');
```

Comment:
```suggestion
	const loggingEnabled = await getGlobalOption(Options.DEBUG_LOGGING_ENABLED);
```

But we would need to import `import * as Options from '@/core/storage/options';`

Reply:
Done now in a1101ef